cmake/Dependencies.aarch64-qnx.cmake

```diff
@@ -29,24 +29,18 @@ endif()
 
 set(CUDA_TOOLKIT_ROOT_DIR ${CUDA_HOST})
 set(CUDA_TOOLKIT_TARGET_DIR ${CUDA_TARGET})
-set(CUDA_USE_STATIC_CUDA_RUNTIME OFF) # "QNX does not have librt.so"
 
 find_package(CUDA 10.0 REQUIRED)
 
-# find_library(CUDA_CUDART_LIBRARY cudart
-#   PATHS ${CUDA_TOOLKIT_TARGET_DIR}
-#   PATH_SUFFIXES lib64 lib)
-
 message(STATUS "Found cudart at ${CUDA_LIBRARIES}")
 
-# set(CUDA_TOOLKIT_ROOT_DIR_INTERNAL ${CUDA_TOOLKIT_ROOT_DIR} CACHE PATH "Cuda toolkit internal root location")
-# set(CUDA_TOOLKIT_TARGET_DIR_INTERNAL ${CUDA_TOOLKIT_TARGET_DIR} CACHE PATH "Cuda toolkit target location")
-
-# set(CUDA_LIBRARIES ${CUDA_TOOLKIT_TARGET_DIR}/lib)
-# set(CUDA_USE_STATIC_CUDA_RUNTIME OFF CACHE BOOL "QNX does not have librt.so")
+if (CUDA_USE_STATIC_CUDA_RUNTIME)
+  list(APPEND DALI_LIBS ${CUDA_LIBRARIES})
+else()
+  list(APPEND DALI_LIBS ${${CUDA_CUDART_LIBRARY}})
```

Comment:
2 queries:
1. Where is CUDA_CUDART_LIBRARY being populated from? find_library is masked out anyway.
2. Why a check for CUDA_USE_STATIC_CUDA_RUNTIME. We know that QNX does not have librt.so. libc.so supplants it.

Reply:
CUDA_CUDART_LIBRARY is found by FindCUDA
CUDA_USE_STATIC_CUDA_RUNTIME is set during FindCUDA based on if static runtime can be used. I know that we will get an OFF here. I thought this would be more akin to native version.